lumicks/pylake/force_calibration/tests/test_exclusions.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+import numpy as np
+from lumicks.pylake.force_calibration.power_spectrum import PowerSpectrum
+
+
+@pytest.mark.parametrize(
+    "exclusion_ranges, result_frequency, result_power",
+    [
+        [[], [1, 3, 5, 7, 9, 11], [10, 30, 50, 70, 90, 110]],  # Empty exclusion range
+        [[(3, 5)], [1, 5, 7, 9, 11], [10, 50, 70, 90, 110]],
+        [[(3, 6)], [1, 7, 9, 11], [10, 70, 90, 110]],
+        [[(3, 6), (3, 6)], [1, 7, 9, 11], [10, 70, 90, 110]],  # Same exclusion range twice
+        [[(5, 6), (3, 8)], [1, 9, 11], [10, 90, 110]],  # Overlapping range
+        [[(3, 8), (5, 6)], [1, 9, 11], [10, 90, 110]],  # Overlapping range (different order)
+        [[(3, 6), (9, 10)], [1, 7, 11], [10, 70, 110]],  # Disjoint range
+    ],
+)
+def test_exclusions(exclusion_ranges, result_frequency, result_power):
```

Comment:
I would sort of expect this test to appear in `test_power_spectrum.py` since it's part of the `PowerSpectrum` class.

Reply:
Yeah, good point, I'll move it.